software-containers/README.md

```diff
@@ -16,3 +16,156 @@ group:      sss files
 
 ### xterm
 `xterm` must be installed on the container
+## Creating a CASA container
+This section describes the process of creating a CASA container. The currently supported CASA containers are in the arcade-casa folder. The process iterates over build, deploy and test.
+### Building A Container
+To build a container, go into the relevant folder with the build files or create a new folder if necessary. There are two main files to building a container: Makefile and Dockerfile.
+#### Makefile
+This is the main file that controls the build process. This file is updated to add/change/remove a CASA version or to change the build process. It invokes docker to build the container.
+#### Dockerfile
+This file controls the building of the CASA container. For example, for versions 4.3.x to 4.4.x, the ADMIT package is not added to the container. Changes to the container are made in this file.
```

Comment:
I think you should omit these three sections (building a container, makefile, dockerfile) and just keep brief instructions on building in your 'how to build' section.  there's too much repitition.

Reply:
Done